data/allPossibleCategories.ts

```diff
@@ -0,0 +1,37 @@
+module.exports = function () {
+	// const allPossibleCategoriesRaw = require('@joplin/lib/pluginCategories.json');
+
+	// interface Category {
+	// 	name: string
+	// }
+
+	// allPossibleCategories will be obtained from @joplin/lib in the next step
+	const allPossibleCategoriesRaw = [
+		'Appearance',
+		'Developer Tools',
+		'Productivity',
+		'Themes',
+		'Integrations',
+		'Viewer',
+		'Search',
+		'Tags',
+		'Editor',
+		'Files',
+		'Personal Knowledge Management',
+	]
+
+
+	const allPossibleCategories = allPossibleCategoriesRaw.map((category) => {
+		return {
+			name: category.toLowerCase().replace(/[ ]/g, '-'),
```

Comment:
```suggestion
			name: category.toLowerCase().replace(/\s/g, '-'),
```
I think matching all space characters with `\s` makes more sense here. Because we already know the category names, feel free to ignore this change!